nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -7,51 +7,51 @@
 - **This will be the last release of Nixpkgs to support macOS Sierra 10.12 to macOS Catalina 10.15.**
   Starting with release 25.05, the minimum supported version will be macOS Big Sur 11, and we cannot guarantee that packages will continue to work on older versions of macOS.
   Users on old macOS versions should consider upgrading to a supported version (potentially using [OpenCore Legacy Patcher](https://dortania.github.io/OpenCore-Legacy-Patcher/) for old hardware) or installing NixOS.
-  If neither of those options are viable and you require new versions of software, [MacPorts](https://www.macports.org/) supports back to Mac OS X Snow Leopard 10.6.
+  If neither of those options are viable and you require new versions of software, [MacPorts](https://www.macports.org/) supports versions back to Mac OS X 10.6 Snow Leopard.
```

Comment:
FWIW this was intentional for consistency with the other releases mentioned in the note and with how Apple usually describes contemporary OS releases, though they’ve been pretty inconsistent historically.

Reply:
OS X to my knowledge has always followed this sort of version scheming (see the [wikipedia article](https://en.wikipedia.org/wiki/Mac_OS_X_Snow_Leopard) for example), so I figured it would be better to stay consistent with that. Not really sure how to refer to macOS releases though, since yeah like you said it's inconsistent; it's probably best to just leave those

Reply:
So Apple almost universally used “v10.6” rather than “10.6” back then from what I’ve seen, but the ordering of the names and version varied a bit, albeit with the name‐first form being predominant in the relatively rare cases where Apple used both. You can see “Mac OS X Snow Leopard v10.6” in the press releases [announcing its release](https://www.apple.com/newsroom/2009/06/08Apple-Unveils-Mac-OS-X-Snow-Leopard/) and [the release of Safari 4](https://www.apple.com/newsroom/2009/06/08Apple-Releases-Safari-4-The-Worlds-Fastest-Most-Innovative-Browser/), but “Mac OS X v10.6 Snow Leopard” on the [technical specifications page](https://support.apple.com/en-gb/112591). The link in the footer of the Apple site from back then was “10.6 Snow Leopard”, and the [contemporary version of the technical specs page](https://web.archive.org/web/20090714080723/http://www.apple.com/macosx/specs.html) also talked about “Mac OS X v10.6 Snow Leopard”. The [Mac OS X Lion installer](https://support.apple.com/en-gb/106383) and [Mac App Store update](https://support.apple.com/en-gb/106470) system requirements list “OS X Snow Leopard 10.6” (anachronistically without the “Mac”, and the Lion installer inconsistently lists the version first in the text description – so perhaps this is due to some later format normalization). This list of [security updates from 2011 to 2012](https://support.apple.com/en-gb/101444) talks about “OS X Lion v10.7.2” and so on, though not for Snow Leopard itself.

On the other hand, for contemporary versions, the “macOS Sequoia 15” format is seemingly almost universally used when Apple (rarely) use both the name and version, e.g. [in the user guide](https://support.apple.com/en-gb/guide/mac-help/apd07d671600/mac), the [SDK release notes](https://developer.apple.com/documentation/macos-release-notes/macos-15-release-notes), and the [security release overview](https://support.apple.com/en-us/100100).

In general Apple didn’t pair the code names and releases as often back then, so it’s harder to point to a canonical format. I went for consistency with the versions people actually care about and Apple’s current usage here. Anyway all of this is of course completely irrelevant and you can do whatever here, but I lost some 10 minutes researching this when I was touching up the wording of the message in the first place so I figured I can at least record my findings for posterity in the unlikely case that this ever comes up again in the future.